### ISSUE TITLE
Run on Android?

### ISSUE THREAD

**Issue by LeoPassos** [enzienaudio/hvcc#36]
Is it possible to develop a game in Unity to run on Android with this project?

**Comment by CarloCattano** [comment]
Yes, it's explained in the docs. Using ndk to compile the plugin
